fdbserver/GlobalTagThrottler.actor.cpp

```diff
@@ -663,15 +755,31 @@ bool rateIsNear(GlobalTagThrottler& globalTagThrottler, TransactionTag tag, Opti
 			rate = it2->second.tpsRate;
 		}
 	}
-	TraceEvent("GlobalTagThrottling_RateMonitor")
+	TraceEvent("GlobalTagThrottling_ClientRateMonitor")
 	    .detail("Tag", tag)
 	    .detail("CurrentTPSRate", rate)
 	    .detail("ExpectedTPSRate", expected);
-	if (rate.present()) {
-		return expected.present() && abs(rate.get() - expected.get()) < 1.0;
-	} else {
-		return !expected.present();
-	}
+	return isNear(rate, expected);
+}
+
+bool statusIsNear(GlobalTagThrottler const& globalTagThrottler,
+                  TransactionTag tag,
+                  double expectedDesiredTps,
+                  Optional<double> expectedLimitingTps,
+                  double expectedTargetTps,
+                  double expectedReservedTps) {
```

Comment:
Nit: use an expected `Status` object so that in the future if we need to add a field in the Status, we don't have to change it here. It's also good to add a `isNear(Status, Status)` function.